### ISSUE TITLE
[Website] "Manage your firefox account" link is broken

### ISSUE THREAD

**Issue by dholbert** [mozilla/fx-private-relay#220]
STR:
 1. Sign in at https://relay.firefox.com/
 2. Click your profile picture at top right, to open a little menu
 3. In that menu, click "Manage your Firefox Account"

ACTUAL RESULTS:
I'm taken to https://profile.accounts.firefox.com/settings, which is just some broken JSON:
```
{"code":404,"errno":999,"error":"Not Found","message":"Not Found"}
```

EXPECTED RESULTS:
I should probably be taken to https://accounts.firefox.com/settings instead (which is a real page)

**Comment by groovecoder** [comment]
Oops. I always forget that production FXA server doesn't use the profile domain for the settings page.

**Comment by sv-hyacoub** [comment]
Verified as fixed on MacOS 10.15, Windows 10 x64 and on Ubuntu 18.04 on Firefox 76.0.1.
